app/models/manageiq/providers/hawkular/inventory/availability_updates.rb

```diff
@@ -0,0 +1,17 @@
+module ManageIQ::Providers
+  class Hawkular::Inventory::AvailabilityUpdates
+    delegate :select, :to => :@targets
+
+    def initialize(targets)
+      @targets = targets
+    end
+
+    def name
+      "Collection of availabilities to update on inventory entities"
+    end
+
+    def id
+      "Collection: #{@targets.map(&:manager_ref)}"
```

Comment:
It doesn't need to be long. But, as `name`, it's only used when [writing logs here](https://github.com/ManageIQ/manageiq/blob/5edac5566820d4d4b14bf55bdcc75b1daca693e5/app/models/ems_refresh/refreshers/ems_refresher_mixin.rb#L23). I found no other usage of it. I'm trying to be verbose for logging.